magento/magento2
# Newsletter subscription emails should also use paragraph tags

### Description (*)

Other emails (also subscription confirmation email in the same module) are using proper html tags for their content.
```
<p>{{trans "To begin receiving the newsletter, you must first confirm your subscription by clicking on the following link:"}}</p>
```
The same should apply for newsletter subscription and unsubscription emails.

This will help in keeping same email styling.

### Related Pull Requests


### Fixed Issues (if relevant)

Couldn't find issue.

### Manual testing scenarios (*)

1. As a logged in user go to User Area - Newsletter section and subscribe / unsubscribe to the newsletter.
2. Go to your email box and check those emails

### Questions or comments


### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#28166: Newsletter subscription emails should also use paragraph tags
